packages/circle-react-elements/package.json

```diff
@@ -47,6 +47,10 @@
   },
   "dependencies": {
     "@hookform/resolvers": "^3.10.0",
+    "@radix-ui/react-dialog": "^1.1.2",
+    "@radix-ui/react-label": "^2.1.0",
+    "@radix-ui/react-select": "^2.1.2",
+    "@radix-ui/react-slot": "^1.1.0",
```

Comment:
I moved radix-ui to a direct dependency since consumers of react-elements already have a long list of peer dependencies to install.